adapters/richaudience/richaudience_test.go

```diff
@@ -46,7 +46,7 @@ type richaudienceSite struct {
 
 func TestJsonSamples(t *testing.T) {
 	bidder, buildErr := Builder(openrtb_ext.BidderRichaudience, config.Adapter{
-		Endpoint: "http://ortb.richaudience.com/ortb/?bidder=pbs",
+		Endpoint: "https://ortb.richaudience.com/ortb/?bidder=pbs",
```

Comment:
We typically advise using a fake endpoint here so you don't have to update all of your tests every time you update your real endpoint.

Reply:
Hi @bsardo 
Thank you for this contribution, for the next changes we will take that into consideration.
But you think that can you approve the change?
I'm waiting for your feedback.
Cheers